servicemesh-install/topics/install.adoc

```diff
@@ -1,21 +1,69 @@
 [[installing-service-mesh]]
 = Installing the {ProductName}
 
-Installing the {ProductShortName} involves creating a custom resource definition file, then installing the operator to create and manage the custom resource.
+Installing the {ProductShortName} involves installing the operator, and then creating and managing a custom resource definition file to deploy the control plane.
 
 [NOTE]
 ====
-Starting with {ProductName} {ProductVersion}, Mixer’s policy enforcement is disabled by default, but you must enable it to run policy tasks. See https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#update-mixer-policy-enforcement[Update Mixer policy enforcement] for instructions on enabling Mixer policy enforcement.
+Starting with {ProductName} 0.9.TechPreview, Mixer’s policy enforcement is disabled by default, but you must enable it to run policy tasks. See https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#update-mixer-policy-enforcement[Update Mixer policy enforcement] for instructions on enabling Mixer policy enforcement.
 ====
 
+[[operator-definition]]
+== What is an operator?
+An _operator_ is a piece of software that enables you to implement and automate common activities, such as installation and configuration, in your Kubernetes cluster. It acts as a controller, allowing you to set or change the desired state of objects within your cluster.
+
+[NOTE]
+====
+{ProductName} {ProductVersion} introduces a modified Operator and a significantly changed installation custom resource file. You can view https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#appendix_B[examples of the custom resource file from previous releases in the appendix].
```

Comment:
Never mind, I just saw Kevin's e-mail comment that "Both are supported in this release, the old one stays around for those who cannot move yet or as a fall back in case we hit some issues."

Reply:
The custom resource changes in this PR are the new way of managing an Istio installation. However, we are keeping backwards compatibility with the old custom resource for the moment as a fallback. If the operator changes fail, the user still has the option to fall back to using the old custom resource which will trigger the ansible installer.